src/CommunityToolkit.Maui/HandlerImplementation/Popup/Popup.shared.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommunityToolkit.Core.Handlers;
+using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Views;
+
+public partial class Popup
+{
+	/// <summary>
+	/// 
+	/// </summary>
+	public static CommandMapper<IPopup, PopupViewHandler> ControlPopUpCommandMapper = new(PopupViewHandler.PopUpCommandMapper)
+	{
+#if IOS
```

Comment:
Probably MacCatalyst as well

Reply:
Just figured out that you discovered why the popup didn't show for MacCatalyst. Great catch!